definitions/divorce/json/CaseEvent.json

```diff
@@ -715,11 +715,11 @@
   {
     "LiveFrom": "02/01/2017",
     "CaseTypeID": "DIVORCE",
-    "ID": "transferCaseFromCTSCpronounce",
+    "ID": "transferCaseFromCTSCAosSolicitor",
     "Name": "Transfer from CTSC to RDC",
     "Description": "Transfer from CTSC to an RDC",
-    "DisplayOrder": 11,
-    "PreConditionState(s)": "AwaitingPronouncement",
+    "DisplayOrder": 19,
```

Comment:
I think the display order should be 3 here as there are only 2 other events with preconditionstate being "Awaiting AOS Solicitor".

Reply:
ok, it was "after 18" in AC.

Reply:
Oow okay, my bad sorry :)